spec/controllers/faye_controller_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+describe FayeController do
+  include Devise::TestHelpers
+  before :each do
+    user = create :user, admin: true
```

Comment:
Devise `master` has `Devise::Test::IntegrationHelpers`, but don't know when there will be a gem release.